### PR TITLE
feat(Design system audits): Update tokens with Shuju

### PR DESCRIPTION
**Pull request by yhamry**
We were sitting and updating the tokens :
Also there's https://github.com/swimmio/swimm/pull/10544 this one after this is published we'll update it

### CODE REVIEW

File: docs/.vuepress/styles/index.scss [hunk 1]
```diff
@@ -1,6 +1,6 @@
 :root,
 html.dark {
-  --c-brand: var(--color-primary-default);
+  --c-brand: var(--color-brand);
```
Comment (talkor) [inline docs/.vuepress/styles/index.scss:3]: just curious, why are we renaming it?

File: public/styles/colors.css [hunk 2]
```diff
@@ -39,7 +39,7 @@
 
     --high-violet: #c558f5;
 
-    --wave-jammer: #a31177;
+    --wave-jammer: #600A6A;
```
Comment (talkor) [inline public/styles/colors.css:42]: I think prettier will change it to lowercase, wouldn't it?

**Comment by talkor** [comment]
@yhamry notice that this PR have breaking changes since we need to update the changed colors in Swimm, so your commit should include indication for breaking change so the version will be upgraded to a new major version. See [here](https://app.swimm.io/workspaces/aRvMqc0yWAVcJlLN944D/repos/pw8tVz0TAgLaoHABDlsw/branch/main/docs/lnjwx)

**Review by talkor — APPROVED** [review]
See comments

File: docs/.vuepress/styles/index.scss (hunk 1 above)
Comment (yhamry) [reply docs/.vuepress/styles/index.scss:3]: Shuju asked to

File: public/styles/colors.css (hunk 2 above)
Comment (yhamry) [reply public/styles/colors.css:42]: it didnt, but I will

**Comment by yhamry** [comment]
@talkor - How's the latest commit?

**Pull request merged**